packages/frontmatter/src/FrontmatterBlock.tsx

```diff
@@ -289,7 +289,7 @@ export function FrontmatterBlock({
     subject || github || venue || biblio || open_access || license || hasExports || isJupyter;
   const hasDateOrDoi = doi || date;
   return (
-    <>
+    <div className="mb-8">
```

Comment:
May I just ask about this `mb-8` here? It's giving me `2rem` extra space that wasn't there before -- between the title and content, or else just pushing the content down when there's no title.

If I remove the `mb-8` class everything looks perfect.

Reply:
Ah, yes, if you only have a title this isn't going to be good. I was testing with a full frontmatter, and that should be on conditionally (or something). Good catch!

We want these conditionally if there is frontmatter. Or maybe better is to override the `h1` here and have an explicit `mb-0` and then have some margin around the stuff that comes after it which is conditionally rendered.

Are you in a place to open a PR for this? If not, I can fix my bug shortly!

The two green things stack, and they should collapse if there isn't anything in between.

![image](https://github.com/executablebooks/myst-theme/assets/913249/419042ce-8199-45d1-9b7a-0e4c41e236c6)

Reply:
Hm I guess I need to think about it a bit. You may get there faster than me.

Reply:
Yeah, I am not quite sure which solution is best. I will try and get to it by the end of today and tag you! Thanks again for pointing it out. :)